Sources/Plasma/NucleusLib/pnNetBase/pnNbError.h

```diff
@@ -112,9 +112,6 @@ enum ENetError {
     // Be sure to add strings for additional error codes in pnNbError.cpp
     // (don't worry, the compiler will tell you if one is missing ;)
     kNumNetErrors,
-
-    // Net messages require ENetError to be sizeof(uint32_t)
-    kNetErrorForceDword             = (uint32_t)(-1)
 };
 
 //COMPILER_ASSERT_HEADER(pnNbError, sizeof(ENetError) == sizeof(uint32_t));
```

Comment:
It may be worth re-enabling this as a static assert to ensure this remains true:

```cpp
static_assert(sizeof(ENetError) == sizeof(uint32_t), "ENetError must be sizeof uint32_t");
```

Reply:
Nice catch. I missed that commented out assert.

I added the static assert in your comment, and verified that without this change that static assert fails on Mac/Clang/64 bit.